src/Summoner/Template.hs

```diff
@@ -433,7 +434,7 @@ createProjectTemplate ProjectData{..} = Dir (toString repo) $
         |]
       where
         githubLine :: Text = memptyIfFalse github $ "The change log is available [on GitHub][2]."
-        githubFootNote :: Text = memptyIfFalse github $ 
```

Comment:
So many spaces I left. What tool show them to you?

Reply:
@willbasky I've set up my emacs to remove this for me automatically.